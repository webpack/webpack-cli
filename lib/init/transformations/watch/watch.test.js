"use strict";

const defineTest = require("../../../utils/defineTest");

defineTest(__dirname, "watch", "watch-0", true, "init");
defineTest(__dirname, "watch", "watch-0", false, "init");
defineTest(__dirname, "watch", "watch-1", true, "init");
defineTest(__dirname, "watch", "watch-1", false, "init");
