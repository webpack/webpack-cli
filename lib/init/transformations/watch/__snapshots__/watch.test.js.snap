// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`watch transforms correctly using "watch-0" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watch: true
}
"
`;

exports[`watch transforms correctly using "watch-0" data 2`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watch: false
}
"
`;

exports[`watch transforms correctly using "watch-1" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watch: true
}
"
`;

exports[`watch transforms correctly using "watch-1" data 2`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watch: false
}
"
`;
